sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -901,6 +901,17 @@ object RapidsConf {
       .booleanConf
       .createWithDefault(false)
 
+  val ENABLE_JSON = conf("spark.rapids.sql.format.json.enabled")
+    .doc("When set to false disables all json input and output acceleration. " +
+      "(only input is currently supported anyways)")
+    .booleanConf
+    .createWithDefault(true)
```

Comment:
These should not be true by default. There are just too many problems with the parsing support right now for us to have them be true.

Reply:
Done